exchange/bitswap/bitswap.go

```diff
@@ -372,16 +372,6 @@ func (bs *Bitswap) ReceiveMessage(ctx context.Context, p peer.ID, incoming bsmsg
 		return
 	}
 
-	// quickly send out cancels, reduces chances of duplicate block receives
-	var keys []*cid.Cid
```

Comment:
@whyrusleeping this code was dead. We're currently sending out cancels from within sessions. However, this *does* mean that the above comment is no longer correct (it'll likely take a bit longer to send out cancels).